server/lib/src/server/mod.rs

```diff
@@ -793,6 +794,15 @@ pub trait QueryServerTransaction<'a> {
             })
     }
 
+    fn get_domain_ldap_allow_unix_pw_bind(&mut self) -> Result<bool, OperationError> {
+        let res = self.internal_search_uuid(UUID_DOMAIN_INFO)?;
+        match res.get_ava_single(Attribute::DomainLdapAllowUnixPwBind) {
```

Comment:
Change to
```
self.internal_search_uuid(UUID_DOMAIN_INFO)
    .map(|entry| {
        entry.get_ava_single_bool(Attribute::DomainLdapAllowUnixPwBind).unwrap_or(true)
    })
```

Reply:
why tho? can we have many entries? I don't understand why there might be differentiating entries where each define changes to a subset of variables? Does each schema become one entry value?

Reply:
Also what if internak_search_uuid gives out an error?

Reply:
That's handled. self.internal_search_uuid() -> returns a Result<Entry, Error>
.map -> for the value of the Ok(entry), get the attribute assuming it's a single-boolean. 
   unwrap_or tream means "if no attribute matechs, or it's the wrong type, return true"

So this handles those cases.